core/Api/UserApi.php

```diff
@@ -258,4 +258,52 @@ public function account_list($user_id, $query = array())
     $filter['created_by'] = $user_id;
     return Account::search($filter);
   }
+
+  /**
+   * Import new users from into DB
+   *
+   * @url GET /users/csv
```

Comment:
Is it not supposed to be a POST API?
@url POST /users/csv

Reply:
Update on latest commit.